sql/core/src/main/scala/org/apache/spark/sql/execution/ui/ExecutionPage.scala

```diff
@@ -148,6 +155,8 @@ class ExecutionPage(parent: SQLTab) extends WebUIPage("execution") with Logging
   }
 
   private def modifiedConfigs(modifiedConfigs: Map[String, String]): Seq[Node] = {
+    if (Option(modifiedConfigs).exists(_.isEmpty)) return Nil
```

Comment:
Why not just 
```
if (modifiedConfigs.isEmpty) ...
```

Reply:
because `modifiedConfigs` might be `null` (https://github.com/apache/spark/pull/35972#discussion_r835124708).

Reply:
yeah, it happens if history server load a old version event log